NON_CONTAINERIZED_DEV_README.md

```diff
@@ -1,49 +1,97 @@
-# Developing on SCP without a Docker container
+# Developing on SCP natively, without a Docker container
 
-Developing on SCP without a Docker container, while less robust, opens up some faster development paradigms, including 
-live css/js reloading, faster build times, and byebug debugging in rails.
+Developing on SCP natively, i.e. outside a Docker container, is less robust but faster.  In native mode, full pages load are 4-7x faster, tests run faster, and rich [byebug debugging](https://github.com/deivid-rodriguez/byebug/blob/master/GUIDE.md) in Rails is possible.
 
 ## SETUP
+Commands below assume your CPU is Apple Silicon / M1, not Intel.
+1. Append `CFLAGS="-Wno-error=implicit-function-declaration"` to your `~/.zshrc` (if using Z shell macOS default in Terminal, or `~/.bash_profile` if Bash), then run `source ~/.zschrc`
```

Comment:
Let's note that this CFLAGS declaration is specific to the bson_ext issue (if we see other build issues, it might be good to know that this one is not a general use environment setting when troubleshooting)